tools/dftgwbse_CO_geoopt/CO.xyz

```diff
@@ -1,4 +1,4 @@
 2
-CO man
-C 0.0 0.0 0.0
-O 1.2 0.0 0.0
+CO
+C -0.0254018 -6.69351e-08 8.88366e-06
+O 1.2254 6.69351e-08 -8.88366e-06
```

Comment:
Why the change?

Reply:
I want to used the optimized geometry to reduce cycles. But anyway, I reduce the optimization cycles to 1. Shall I go back to the previous geometry?

Reply:
Does it take that long, I wanted to have 2-3 steps so that we see sth.